lib/Backend/AsmJsJITInfo.h

```diff
@@ -18,11 +18,11 @@ class AsmJsJITInfo
     int Get##name##TmpCount() const     { return m_data.typedSlotInfos[WAsmJs::##type].tmpCount; }\
     int Get##name##VarCount() const     { return m_data.typedSlotInfos[WAsmJs::##type].varCount; }
 
-    TYPED_SLOT_INFO_GETTER(Double, FLOAT64);
-    TYPED_SLOT_INFO_GETTER(Float, FLOAT32);
-    TYPED_SLOT_INFO_GETTER(Int, INT32);
-    TYPED_SLOT_INFO_GETTER(Int64, INT64);
-    TYPED_SLOT_INFO_GETTER(Simd, SIMD);
```

Comment:
>); [](start = 37, length = 2)

nit: I think I prefer it with the semicolon for consistency so it looks more like a declaration